docs/development.md

```diff
@@ -266,6 +266,19 @@ This will mount your local build of Faasm inside a local cluster defined in
 Before doing so, make sure you've completely removed any existing faasm
 containers.
 
+Docker compose is well-suited for development as it allows to patch many aspects
+of your cluster without ever shutting down the deployment (which you may do
+by running `docker-compose down`).
+
+### Scaling services
+
+To scale up or down a service you may run:
+```
+docker-compose up -d --scale <service>=<NUM_REPLICAS> [--force-recreate/no-recreate]
+```
```

Comment:
This is just repeating the docker-compose documentation. The only service we might want to scale is the worker, so you could change this section to something like "Running with multiple workers"